plone/app/users/browser/register.py

```diff
@@ -302,7 +295,7 @@ def validate_registration(self, action, data):
 
         # passwords should match
         if 'password' in form_field_names:
-            assert('password_ctl' in form_field_names)
+            assert 'password_ctl' in form_field_names
```

Comment:
Usage of ``assert`` outside of tests is, in general, a bad idea. It does get removed in optimized mode (``python -O``). Well, this was not introduced by this PR.